ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -124,17 +141,5 @@ def _get_explicit_node_display_attr(
     def __init_subclass__(cls, **kwargs: Any) -> None:
         super().__init_subclass__(**kwargs)
 
-        original_base = get_original_base(cls)
-        node_class = get_args(original_base)[0]
-        if isinstance(node_class, TypeVar):
-            bounded_class = node_class.__bound__
-            if inspect.isclass(bounded_class) and issubclass(bounded_class, BaseNode):
-                cls._node_display_registry[bounded_class] = cls
-        elif issubclass(node_class, BaseNode):
-            if has_wrapped_node(node_class):
-                wrapped_node = get_wrapped_node(node_class)
-                if wrapped_node._is_wrapped_node:
-                    cls._node_display_registry[wrapped_node] = cls
-                    return
```

Comment:
we ended up not needing this logic. I know this had something to do with mapping TryNode's adorned nodes to the try node's display, but that always felt risky. I think some other changes below relieve this. Ideally we could keep this simple - the registry is simply a map between the parameterized type and the display class